proj/src/main/webapp/script.js

```diff
@@ -117,15 +117,15 @@ function appendMessage(payload){
  */
 
 const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
-const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
+const PATH = '/chat/test/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
 const LIMIT = 20; // how many messages to load at a time
 var firstChildKey;
 
 function init() {
     initRef();
     clickWithEnterKey();
 
-    const chat = document.getElementById('chat-as-list');
+    const chat = document.querySelector('.chat');
```

Comment:
I think if there is only one item, maybe getting it by Id will be better. If there are multiple chat divs on the page will you need to add this to each one? If so, you might need a loop below.

Reply:
since I don't think we know yet if we're going to display multiple chats at the same time, i just gave the one we have an ID and changed it to getElementByID instead